### DISCUSSION TITLE
Does cri-o need access to firewall to add more rules?

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#7849] [Q&A]
How to set firewall access to cri-o?

**Comment by haircommander** [comment]
I am pretty sure this would be a configuration for the firewall itself, unless you're asking about containers accessing outside of the firewall, in which case it would be a configuration on the CNI + firewall I think

**Reply by alpana17** [reply]
As in docker we can enable or disable docker daemon to make changes to iptables
dockerd --iptables=false

**Reply by alpana17** [reply]
Is there something similar in cri-o?

**Reply by haircommander** [reply]
this should be handled by CNI I think. Do you see CRI-O making changes to iptables?

**Comment by alpana17** [comment]
Till now, I have not seen any changes in my iptables list
It might be because, I just installed cri-o and started the service
I have not launched any containers with it
